outputs.tf

```diff
@@ -1,4 +1,4 @@
 output "ecr_repository_urls" {
-  value       = aws_ecr_repository.repos.*.repository_url
+  value       = values(aws_ecr_repository.repos)[*].repository_url
   description = "A list of urls of the repositories that were created."
-}
```

Comment:
I'm assuming this results in the same output? What is the point of this change?

Reply:
it doesn't seem to work anymore without the changes, yes the output is still:
```
    [
      + "309691092939.dkr.ecr.us-east-1.amazonaws.com/api",
      + "309691092939.dkr.ecr.us-east-1.amazonaws.com/cognito",
      + "309691092939.dkr.ecr.us-east-1.amazonaws.com/datalog",
      + "309691092939.dkr.ecr.us-east-1.amazonaws.com/db-schema",
      + "309691092939.dkr.ecr.us-east-1.amazonaws.com/email",
    ]
```